.github/labeler.yml

```diff
@@ -0,0 +1,23 @@
+# Add affects/core label to all changes to the core package; exclude changes to test files
+affects/core:
+  - any: [ '/*.go', '!/*_test.go']
+
+# Add affects/services label to all changes to the services directory
+affects/services:
+  - 'services/*'
+
+# Add affects/tests label when any test file is changed
+affects/tests:
+  - '**/*_test.go'
+
+# Add affects/docs label when any markdown file or the docs directory is changed
+affects/docs:
+  - 'docs/**/*'
+  - '/*.md'
+
+# Add affects/ci label when any CI related file is changed
+affects/ci:
+  - '.github/**/*'
```

Comment:
If the syntax `**/` does not match files too, but only directories, maybe we should add another one for the files under `.github/` (Currently [PULL_REQUEST_TEMPLATE.md](https://github.com/nikoksr/notify/blob/main/.github/PULL_REQUEST_TEMPLATE.md) and [config.yml](https://github.com/nikoksr/notify/blob/main/.github/config.yml)).